.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+on: [push, pull_request]
+permissions: {}
+
+jobs:
+  trigger:
+    if: |
+      github.event_name == 'push' ||
+      (
+        github.event_name == 'pull_request' &&
+        github.event.pull_request.head.repo.full_name != github.event.pull_request.base.repo.full_name
```

Comment:
What does this line do?

Reply:
Or more, why is it needed?

Reply:
The problem it solves is that we want to run this CI when you have a push to the repository, but also when you have PR from the fork. But you don't want to run "pull_request" pipelines from internal PRs, because then you have two pipelines running in parallel: triggered by push and triggered by pull_request event.